### DISCUSSION TITLE
Allow DELETE method for API node

### DISCUSSION THREAD

**Discussion by mbartylak** [zachowj/node-red-contrib-home-assistant-websocket#1435] [Feature Request]
The API node currently allows only for GET and POST. There are some use cases where DELETE would also be useful and it seems like a very easy thing to add. Theoretically this could be the case for PUT as well.

My use case:
I have a flow which dynamically creates light groups via config_entries API, based on device names assigned to individual light bulbs. For example bulbs "office_bulb_1", "office_bulb_2" will automatically be added to an "office_lamp" group. I can create and edit such groups using POST calls but I'm not able to delete them programmatically from node red as DELETE is not an available method.

**Comment by zachowj** [comment]
Added 7bba5af90238001e3120117d54ac0fab5fdfb1b6
